biothings_annotator/application/views/annotator.py

```diff
@@ -14,6 +14,21 @@
 logger = logging.getLogger(__name__)
 
 
+class StatusView(HTTPMethodView):
+    async def get(self, request: Request):
+        curie = "NCBIGene:1017"
+        fields = "_id"
+        raw = bool(int(request.args.get("raw", 0)))
+        include_extra = bool(int(request.args.get("include_extra", 1)))
+
+        annotator = Annotator()
+        try:
+            annotated_node = annotator.annotate_curie(curie, fields=fields, raw=raw, include_extra=include_extra)
+            return sanic.json(annotated_node)
```

Comment:
Don't have to do this now but I imagine for a status route we should have something simpler for establishing "status". I don't know what we do for other status endpoints but this could still fail and just return whatever the response is from the curie annotation. At some point we should add some response verification and return a different structure than the `CurieView` endpoint, but at the moment that likely isn't required

Reply:
I agree, we could define the checks and the response data structure. I'm opened to make the necessary changes.

Reply:
let's make `/status` return a simpler response like our BioThings API's `/status`:

```
{ "success": true}
```
or
```
{"success": false, "error": '...'}
```
when something is wrong, better capture the brief msg, if any.